unit_tests/bit_field_original_value.feature

```diff
@@ -5,24 +5,6 @@ Feature: BitField.original_value
         When Calling original_value
         Then Result equals .render()
 
-    Scenario: Same as initial render after mutation
```

Comment:
Why remove the scenarios but not the implementation? And why remove them at all? Are these tests irrelevant now?
Is it because we no longer have `reset()`? If so there are still some tests using reset.

Reply:
It's because we no longer have reset. And yeah the implementation should be removed also.